docs/multi-organization-guide.md

```diff
@@ -0,0 +1,182 @@
+2. Once the code under development is stable and underwent successful unit and basic integration tests, it is moved from the "dev" AWS Organization to the "int" AWS Organization. The process of propagating code form one AWS Organization to another is described here [Propagating Code Changes Between ADF Installations](#Propagating-Code-Changes-Between-ADF-Installations).
+3. The "int" AWS Organization is used for final integration testing and verification. The "dev" AWS Organization is exclusivly reserved for the landing zone development team. No end-user has access to the dev org.
+4. Once all tests passed successfully, the code is moved from the "int" AWS Organization to the "prod" AWS Organization.
+5. Assuming that the propagation and the deploymemt in the "prod" AWS Organization was successful, the code is now fully deployed in the "prod" AWS Organization and is available to the end-users.
+
+The benefits of such a setup is that an Enteprise can apply the same common `Software Development Lifecycle` to typical "one-off" landing zone services that are hard to test in a single-orgaization setup. It enables an Enterprises 'Cloud Center of Excellence' (landing zone team) a 
+controlled process to develop, test and validate changes to wide reaching mission-critical 
+services, including but not limited to:
+- Service Control Policies changes.
+- Identity Center and IAM based Access Management Configuration changes.
+- Deployment Framework changes.
+- AWS Organization changes; including OU structure.
+- Control Tower and Account Provisioning configurations changes.
+- Centralized security service configuration changes.
+- Centralized cost management configuration changes.
+- Centralized networking changes.
+
+The following sections are written in the context of this "".
+
+## Propagating Code Changes Between ADF Installations 
+With multiple ADF configurations across multiple AWS organization there comes a new challenge to maintain repositories and its configurations across multiple environments. 
+This applies to the following repositories: 
+- aws-deployment-framework-bootstrap
+- aws-deployment-framework-pipelines
+- any other landing zone repository
+
+As the requirements of the bootstrapped resources and adf configuration evolve they need to be updated over time, and these changes ideally propagated from one Installation to the next in a coordinated, controlled fashion.
+
+With an `Environment Branching` approach it's possible to build a 'hands-off' 
+automated mechanism to Promote from a "dev" AWS Organization installation to a "prod" AWS Organization installation.
+This means that for example merging code from the "dev" branch of a repository to the "int" branch of a repository, will trigger the deployment process in the "int" AWS Organization.
```

Comment:
```suggestion
This means that, for example merging code from the "dev" branch of a repository to the "int" branch of a repository, it will trigger the deployment process in the "int" AWS Organization.
```